src/annotations/AnnotationFile.php

```diff
@@ -69,6 +75,23 @@ public function __construct($path, array $data)
         $this->data = $data;
         $this->namespace = $data['#namespace'];
         $this->uses = $data['#uses'];
+
+        foreach ($data['#traitMethodOverrides'] as $class => $methods) {
```

Comment:
For cases, when you have this file parsed & cached using previous parser version such key won't be there and a PHP notice about missing array key would be generated.